resources/views/site/landingPage/_footer----rlc.blade.php

```diff
@@ -0,0 +1,60 @@
+<div class="o-landingpage__prefooter o-blocks rlc">
+    <div class="f-contact m-prefooter-section">
+        <div class="m-prefooter-section-background">
+            <h3 id="contact" class="contact-header title f-module-title-2">
+                {{ $item->labels->get('contact_header') }}
+            </h3>
+            <div class="contact-intro">
+                {!! $item->labels->get('contact_intro') !!}
+            </div>
+            <div class="f-contact-blocks">
+                {!! $item->renderNamedBlocks('contact') !!}
+            </div>
```

Comment:
This is where the contact section ("Get in Touch") is rendered. This set of blocks allows for the newsletter signup block and the paragraph block (see https://github.com/art-institute-of-chicago/artic.edu/pull/205/files#diff-a80a7be499feef237eb842a3d89b78ce6bf066502ff80748109f3b9603821d27R173-R180), but since the paragraph block template can contain arbitrary markup with no wrapping div / container (https://github.com/art-institute-of-chicago/artic.edu/blob/develop/resources/views/site/blocks/paragraph.blade.php#L84), there is no way to tell when one paragraph block ends and the next begins, making it impossible to style.

Reply:
See https://github.com/art-institute-of-chicago/artic.edu/pull/204/files#r1480482050